src/skills-builder/skills-builder-modal/select-preferences/JobTitleAutosuggest.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+} from '@edx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+import { useAlgoliaSearch } from '../../utils/search';
+
+const JobTitleAutosuggest = ({ onSelected }) => {
+  // We only need to destructure the job search index here
+  const [,, jobSearchIndex] = useAlgoliaSearch();
```

Comment:
I assumed we would be passing around the index where needed instead of re-instantiating the objects with an additional call to useAlgoliaSearch. Do we have access to to passing in the index here to this component?

Reply:
Yes we can do this

Reply:
The `useAlgoliaSearch` function is now called inside the `SkillsContextProvider` so that this data is available to all components in the `SkillsBuilder`.